### PR TITLE
Log to stdout on all platforms except Android

### PR DESCRIPTION
**Pull request by cbracken**
Fixes a Fuchsia breakage introduced in
dcfcc9259d4f4321e8daea24f1778694a8dca98a.

### CODE REVIEW

File: lib/ui/dart_runtime_hooks.cc [hunk 1]
```diff
@@ -143,9 +143,9 @@ void Logger_PrintString(Dart_NativeArguments args) {
     // Write to the logcat on Android.
     const char* tag = Settings::Get().log_tag.c_str();
     __android_log_print(ANDROID_LOG_INFO, tag, "%.*s", (int)length, chars);
-#elif defined(__APPLE__) || defined(OS_LINUX)
-    // On iOS and in flutter_tester (on both macOS and Linux, write directly to
-    // stdout. On iOS, this is redirected to ASL via
+#else
+    // On Fuchsia, iOS and in flutter_tester (on both macOS and Linux, write
```
Comment (Hixie) [inline lib/ui/dart_runtime_hooks.cc]: nit: missing close paren

File: lib/ui/dart_runtime_hooks.cc [hunk 2]
```diff
@@ -143,9 +143,9 @@ void Logger_PrintString(Dart_NativeArguments args) {
...
+    // directly to stdout. On iOS, this is redirected to ASL via
     // RedirectIOConnectionsToSyslog in platform_mac.mm.
     // TODO(cbracken) replace with dedicated (non-stdout) logging on iOS.
```
Comment (Hixie) [inline lib/ui/dart_runtime_hooks.cc]: nit: missing colon after `)` in TODO

**Comment by Hixie** [comment]
LGTM

**Comment by Hixie** [comment]
is there any way to test this?

File: lib/ui/dart_runtime_hooks.cc (hunk 1 above)
Comment (cbracken) [inline lib/ui/dart_runtime_hooks.cc]: Done

File: lib/ui/dart_runtime_hooks.cc (hunk 2 above)
Comment (cbracken) [inline lib/ui/dart_runtime_hooks.cc]: Done

**Comment by cbracken** [comment]
Will ask what are options are re: getting a Fuchsia chromebot, and what such a setup would look like.

**Comment by cbracken** [comment]
Fixes flutter/flutter#10509

**Pull request merged**